_author_instructions/01_policies.md

```diff
@@ -103,6 +103,31 @@ However, it is still important that your references use Title Case for article t
 You are responsible for checking that any special characters appear properly, and otherwise ensuring that your final reference list is correct and well-formatted, including sufficient detail.
 Pay particular attention to formatting of non-journal articles as it is easy to end up omitting key citation information for content like software, poster presentations, preprints or other materials.
 
+### Figures
+
+Our largest concern in the construction and preparation of figures is their quality in the final PDF, so we do not *require* any specific image format for figures.
+Rather, figures should:
+- Appear polished in the final PDF of the article, even if zoomed in a great deal
```

Comment:
Possible addition: "We suggest using vector graphics whenever possible for diagrams, that the images are saved at the resolution that they are actually used in the paper, and that images that are bitmap format (such as jpeg/png/giff), are at least 200 dpi." Open for suggestions on the dpi number.